tests/NHSD.BuyingCatalogue.Identity.Api.UnitTests/OrganisationControllerTests.cs

```diff
@@ -64,18 +101,54 @@ public async Task GetAllAsync_SingleOrganisationExists_ReturnsTheOrganisation(st
             var organisationList = organisationResult.Organisations.ToList();
             organisationList[0].Name.Should().Be(name);
             organisationList[0].OdsCode.Should().Be(ods);
+            organisationList[0].PrimaryRoleId.Should().Be(primaryRoleId);
+            organisationList[0].CatalogueAgreementSigned.Should().Be(catalogueAgreementSigned);
+
+            if (hasAddress)
+            {
+                organisationList[0].Address.Should().BeEquivalentTo(_address1);
+            }
+            else
+            {
+                organisationList[0].Address.Should().BeNull();
+            }
         }
 
         [Test]
         public async Task GetAllAsync_ListOfOrganisationsExist_ReturnsTheOrganisations()
         {
-            var org1 = new Organisation(new Guid(), "Organisation 1", "ODS 1");
-            var org2 = new Organisation(new Guid(), "Organisation 2", "ODS 2");
-            var org3 = new Organisation(new Guid(), "Organisation 3", "ODS 3");
+            var org1 = new Organisation
```

Comment:
One suggestion here would be to reduce the amount of code by introducing the builder pattern for the organisation class. This could contain sensible defaults. If we add another property to the organisation class in the future it could potentially effect multiple tests. With the builder pattern it could potentially mitigate this impact with a sensible default.